In [10]:
import matplotlib.pyplot as plt
import pandas as pd
#Reading the data
cwData = pd.read_csv(r"C:\Users\alana\Documents\TC1002S\NotebooksProfessor\datasets\iris\iris.csv", header=None)

In [11]:
#Printing the data set
print(cwData)

       0    1    2    3               4
0    5.1  3.5  1.4  0.2     Iris-setosa
1    4.9  3.0  1.4  0.2     Iris-setosa
2    4.7  3.2  1.3  0.2     Iris-setosa
3    4.6  3.1  1.5  0.2     Iris-setosa
4    5.0  3.6  1.4  0.2     Iris-setosa
..   ...  ...  ...  ...             ...
145  6.7  3.0  5.2  2.3  Iris-virginica
146  6.3  2.5  5.0  1.9  Iris-virginica
147  6.5  3.0  5.2  2.0  Iris-virginica
148  6.2  3.4  5.4  2.3  Iris-virginica
149  5.9  3.0  5.1  1.8  Iris-virginica

[150 rows x 5 columns]


In [12]:
#Print the number of rows
print(len(cwData))

150


In [13]:
#Print the number of columns
print(len(cwData.columns))

5


In [14]:
#Checking out types of variable
cwData.dtypes

0    float64
1    float64
2    float64
3    float64
4     object
dtype: object

1: How many observations (rows) are in total? 150

2: How many variables (columns are in total)? What do they represent? 5 columns, they represent type of flower and quantity in each of the three trees

3: How many observations are in each type of flower? 50 observations

4: What is the type of data for each variable? Float 64 and object

5: What are the units for each variable? Their dtype is object, while in the csv there are floats and object